cmd/id-exporter/main_test.go

```diff
@@ -370,7 +364,7 @@ func (tc testCtx) addCertificates(t *testing.T) {
 		DNSNames:     []string{"example-b.com"},
 		SerialNumber: serial2,
 	}
-	certDerB, _ := x509.CreateCertificate(rand.Reader, &rawCertB, &rawCertB, &testKey.PublicKey, &testKey)
+	certDerB, _ := x509.CreateCertificate(rand.Reader, &rawCertB, &rawCertB, key.Public(), key)
```

Comment:
above on line 340, you made a change to check the error return; here you left it as `_`. Probably we should be consistent one way or the other.